Description: Patch and distribute upstream manpages instead of
 maintaining a seperate set in Debian.
Author: Scott Howard <showard@debian.org>
Index: bitcoin/contrib/debian/manpages/bitcoin.conf.5
===================================================================
--- bitcoin.orig/contrib/debian/manpages/bitcoin.conf.5	2013-04-09 16:28:06.809683732 -0400
+++ bitcoin/contrib/debian/manpages/bitcoin.conf.5	2013-04-09 16:28:06.801683732 -0400
@@ -2,11 +2,11 @@
 .SH NAME
 bitcoin.conf \- bitcoin configuration file
 .SH SYNOPSIS
-All command-line options (except for '-datadir' and '-conf') may be specified in a configuration file, and all configuration file options may also be specified on the command line. Command-line options override values set in the configuration file.
+All command-line options (except for '\-datadir' and '\-conf') may be specified in a configuration file, and all configuration file options may also be specified on the command line. Command-line options override values set in the configuration file.
 .TP
 The configuration file is a list of 'setting=value' pairs, one per line, with optional comments starting with the '#' character.
 .TP
-The configuration file is not automatically created; you can create it using your favorite plain-text editor. By default, bitcoind(1) will look for a file named bitcoin.conf(5) in the bitcoin data directory, but both the data directory and the configuration file path may be changed using the '-datadir' and '-conf' command-line arguments.
+The configuration file is not automatically created; you can create it using your favorite plain-text editor. By default, bitcoind(1) will look for a file named bitcoin.conf(5) in the bitcoin data directory, but both the data directory and the configuration file path may be changed using the '\-datadir' and '\-conf' command-line arguments.
 .SH LOCATION
 bitcoin.conf should be located in $HOME/.bitcoin
 .SH NETWORK-RELATED SETTINGS
@@ -53,9 +53,10 @@
 You can use *bitcoin* or *bitcoind(1)* to send commands to *bitcoin*/*bitcoind(1)* running on another host using this option.
 .TP
 \fBrpcssl=\fR\fI'1'\fR
-Use Secure Sockets Layer (also known as TLS or HTTPS) to communicate with *bitcoin* '-server' or *bitcoind(1)*. Example of OpenSSL settings used when *rpcssl*='1':
+Use Secure Sockets Layer (also known as TLS or HTTPS) to communicate with *bitcoin* '\-server' or *bitcoind(1)*. Example of OpenSSL settings used when *rpcssl*='1':
 .TP
-\fBrpcsslciphers=\fR\fI'TLSv1+HIGH:!SSLv2:!aNULL:!eNULL:!AH:!3DES:@STRENGTH'\fR
+\fB\-rpcsslciphers=\fR<ciphers>
+Acceptable ciphers (default: TLSv1+HIGH:\:!SSLv2:\:!aNULL:\:!eNULL:\:!AH:\:!3DES:\:@STRENGTH)
 .TP
 \fBrpcsslcertificatechainfile=\fR\fI'server.cert'\fR
 .TP
Index: bitcoin/contrib/debian/manpages/bitcoind.1
===================================================================
--- bitcoin.orig/contrib/debian/manpages/bitcoind.1	2013-04-09 16:28:06.809683732 -0400
+++ bitcoin/contrib/debian/manpages/bitcoind.1	2013-04-09 16:28:06.801683732 -0400
@@ -4,7 +4,7 @@
 .SH SYNOPSIS
 bitcoin [options] <command> [params]  
 .TP
-bitcoin [options] help <command> - Get help for a command
+bitcoin [options] help <command> \- Get help for a command
 .SH DESCRIPTION
 This  manual page documents the bitcoind program. Bitcoin is a peer-to-peer digital currency. Peer-to-peer (P2P) means that there is no central authority to issue new money or keep track of transactions. Instead, these tasks are managed collectively by the nodes of the network. Advantages:
 
@@ -75,7 +75,7 @@
 Server private key (default: server.pem)
 .TP
 \fB\-rpcsslciphers=\fR<ciphers>
-Acceptable ciphers (default: TLSv1+HIGH:!SSLv2:!aNULL:!eNULL:!AH:!3DES:@STRENGTH)
+Acceptable ciphers (default: TLSv1+HIGH:\:!SSLv2:\:!aNULL:\:!eNULL:\:!AH:\:!3DES:\:@STRENGTH)
 .TP
 \-?
 This help message
@@ -115,7 +115,7 @@
 Returns boolean true if server is trying to generate bitcoins, false otherwise.
 .TP
 \fBsetgenerate 'generate' ['genproclimit']\fR
-Generation is limited to ['genproclimit'] processors, -1 is unlimited.
+Generation is limited to ['genproclimit'] processors, \-1 is unlimited.
 .TP
 \fBgethashespersec\fR
 Returns a recent hashes per second performance measurement while generating.
@@ -166,7 +166,7 @@
     "confirmations" : number of confirmations of the most recent transaction included.
 .TP
 \fBlisttransactions 'account' ['count=10']\fR
-Returns a list of the last ['count'] transactions for 'account' - for all accounts if 'account' is not specified or is "*". Each entry in the list may contain:
+Returns a list of the last ['count'] transactions for 'account' \- for all accounts if 'account' is not specified or is "*". Each entry in the list may contain:
 
     "category"      : will be generate, send, receive, or move.
     "amount"        : amount of transaction.
